test/server/browser-provider-module-loader-test.js

```diff
@@ -0,0 +1,28 @@
+var BrowserProviderModuleLoader  = require('../../lib/browser/provider/module-loader');
+var expect = require('chai').expect;
+
+describe('Browser provider module loader', function () {
+    describe('module loading regex', function () {
+        it('Should find @private/testcafe-browser-provider-package', function () {
+            expect(new BrowserProviderModuleLoader().isPrivateModule('@private/testcafe-browser-provider-package')).to.be.not.null;
```

Comment:
If you refactor `BrowserProviderModuleLoader` into `getPoviderModuleName`, please rewrite tests in this section to look like the test in the `private module loading package name loading`. Then you can name the combined section as a `Provider module name resolving`.